common/src/IO/ObjSerializer.h

```diff
@@ -118,6 +120,7 @@ namespace TrenchBroom {
             std::ostream& m_objStream;
             std::ostream& m_mtlStream;
             Path m_mtlPath;
+            std::shared_ptr<IO::ObjExportOptions> m_options;
```

Comment:
Missing `memory` include, plus we use `shared_ptr` to model shared ownership, which I don't think you need here. In fact, I'd model this as a reference instead, or pass it by value into the constructor. See comments about modeling export options using `std::variant` above.